Shared/TimeConstants.swift

```diff
@@ -62,38 +62,40 @@ extension Date {
         return Date(timeIntervalSince1970: Double(microsecondTimestamp) / 1000000)
     }
 
-    public func toRelativeTimeString() -> String {
+    public func toRelativeTimeString(dateStyle: DateFormatter.Style = .short, timeStyle: DateFormatter.Style = .short) -> String {
```

Comment:
For the login details view, which has date modified and created, we want the date formatted with a 'medium' style which uses a 3-letter month, instead of a fully numeric mm/dd/yyyy 'short' style.

Reply:
Like so:
<img width="360" alt="image" src="https://user-images.githubusercontent.com/5495083/53763687-67913a00-3e99-11e9-8b10-42ad077feb76.png">